src/theme/foundations/colours.ts

```diff
@@ -30,16 +31,16 @@ export const colours: { [k in IsomerColorScheme]: NestedRecord } = {
     900: "#0C2E6A",
   },
   secondary: {
-    50: "#FBF8FF",
-    100: "#EFE7FF",
-    200: "#C9B3FF",
-    300: "#AA90FA",
-    400: "#8C73DB",
-    500: "#6D58BB",
-    600: "#5543A4",
-    700: "#493897",
-    800: "#3B2E8A",
-    900: "#271E78",
+    50: "#F8F9FA",
```

Comment:
have we checked for visual inconsistencies?

Reply:
locally yes, but I am not the expert on this. 
Once we clarify this implementation with design system, can move forward to test on staging + get OK from design

Reply:
Okay sounds good